agent_plugin_builder/vendor_dirs.py

```diff
@@ -60,71 +87,135 @@ def check_if_common_vendor_dir_possible(build_dir: Path, uid, gid) -> bool:
     windows_packages_path.unlink()
 
     response = all_equal([linux_packages, windows_packages])
+    if response:
+        logger.info("Common vendor directory is possible")
```

Comment:
```suggestion
    if linux_packages == windows_packages:
        logger.info("Common vendor directory is possible")
```

See https://github.com/guardicode/agent-plugin-builder/pull/1/files#r1657406678 for an explanation.

See